src/test/java/com/artipie/nuget/NugetITCase.java

```diff
@@ -118,8 +144,14 @@ private void createNugetConfig() throws Exception {
                 "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n",
                 "<configuration>",
                 "<packageSources>",
-                String.format("<add key=\"%s\" value=\"%s\"/>", source, url),
+                String.format(
+                    "<add key=\"%s\" value=\"%s\"/>",
+                    source, String.format("%s/index.json", this.url)
+                ),
```

Comment:
@genryxy here we specify URL for `"artipie-nuget-test"` alias. But I am not sure we are actually using it anywhere, can't we just remove this `<packageSources>` section?